Soundify/Soundify/Widgets/TopDetailView/TopDetailView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  TopDetailView.swift
+//  Soundify
+//
+//  Created by Viet Anh on 3/15/20.
+//  Copyright © 2020 VietAnh. All rights reserved.
+//
+
+import UIKit
+
+protocol TopDetailViewDelegate: AnyObject {
+    func leftBarButtonItemClicked()
+    func scrollViewDidScroll(_ scrollView: UIScrollView)
+}
+
+class TopDetailView: UIView {
+    
+    //MARK: - Init
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    convenience init(frame: CGRect ,with album: Album) {
+        self.init(frame: frame)
+        setUpDetailView(with: album)
+    }
+    
+    convenience init(frame: CGRect ,with playlist: Playlist) {
+        self.init(frame: frame)
+        setUpDetailView(with: playlist)
+    }
+    
+    private let nibName = "TopDetailView"
+    private func commonInit() {
+        Bundle.main.loadNibNamed(nibName, owner: self, options: nil)
+        addSubview(contentView)
+    }
+
+    //MARK: - IBOutlet & IBAction
+    @IBOutlet private var contentView: UIView!
+    @IBOutlet weak var navigationBarView: UIView!
+    @IBOutlet private weak var albumImageView: UIImageView!
+    @IBOutlet private weak var imageTopSpace: NSLayoutConstraint!
+    @IBOutlet private weak var titleLabel: UILabel!
+    
+    @IBAction private func backButtonClicked(_ sender: UIButton) {
+        delegate?.leftBarButtonItemClicked()
+    }
+
+    //MARK: - Variable for Stretchy Header Effect
+    /// Delegate
+    weak var delegate: TopDetailViewDelegate?
+    
+    /// Public
+    let edgeInsets = UIEdgeInsets(top: 270, left: 0, bottom: 0, right: 0)
+    
+    /// Private
+    private let viewHeight: CGFloat = 270
+    private var isHiddenTitleNavigation = true
+    private let topAlignmentConstraintImageView: CGFloat = 15
+    private let screenHeight = UIScreen.main.bounds.size.height
+    private let screenWidth = UIScreen.main.bounds.size.width
+    private var navigationBarHeight: CGFloat {
+        let window = UIApplication.shared.windows.filter {$0.isKeyWindow}.first
```

Comment:
```suggestion
        let window = UIApplication.shared.keyWindow
```

Reply:
<img width="1037" alt="Screen Shot 2020-03-16 at 10 17 34 PM" src="https://user-images.githubusercontent.com/60588629/76772777-0b633480-67d4-11ea-8afd-708058d2f193.png">
Lúc đầu em cũng để như vậy xong bị báo như này xong em sửa theo ở đây: 
https://stackoverflow.com/questions/57134259/how-to-resolve-keywindow-was-deprecated-in-ios-13-0

Reply:
ok